<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCmpWidget</class>
 <widget class="QWidget" name="FCmpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCmpWidget</string>
  </property>
  <widget class="QTreeView" name="treeView1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>781</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnCompare">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rBtnAll">
      <property name="text">
       <string>All Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnCpp">
      <property name="text">
       <string>C/C++</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnCmake">
      <property name="text">
       <string>CMake (VSCode)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnQt">
      <property name="text">
       <string>Qt (qmake)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnWMfc">
      <property name="text">
       <string>WinAPI/MFC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnCs">
      <property name="text">
       <string>C# (.NET)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnJavaA">
      <property name="text">
       <string>JAVA (Android)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>639</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushBtnAbout">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>720</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
